File: Source Code/Samples/Bölüm 2/İlk Uygulama/Konsol.cs
using System;

namespace DotNetKitabý.ÖrnekUygulamalar
{
    class Program
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Herhangi bir deðer giriniz:");
            string okunanMetin = Console.ReadLine();
            Console.WriteLine(
                string.Format("Tarih-Saat:{0}, Girilen Deðer:{1}",
                    DateTime.Now.ToString(), 
                    okunanMetin));
        }
    }
}

File: Source Code/Samples/Bölüm 2/İlk Uygulama/Konsol.vb
Imports System

Namespace DotNetKitabý.ÖrnekUygulamalar
    Module Program
        Sub Main()
            Dim okunanMetin As String
            Console.WriteLine("Herhangi bir deðer giriniz:")
            okunanMetin = Console.ReadLine()
            Console.WriteLine(String.Format("Tarih-Saat:{0}, Girilen Deðer:{1}", _
                DateTime.Now.ToString(), _
                okunanMetin))
        End Sub
    End Module
End Namespace